### ISSUE TITLE
Discrepancy in ESP8266 FW and LED.py

### ISSUE THREAD

**Issue by otcbl1nk** [not-matt/audio-reactive-led-strip#4]
Looks like with the latest update LED.py is no longer sending valid data for the ESP8266. This is resulting in everything being pretty broken. Might want to update the ESP8266 files to account for your new changes (or switch LED.py back which is what I did to unblock)

**Comment by not-matt** [comment]
Sorry, branch muddle. I'll sort it.

**Comment by otcbl1nk** [comment]
Awesome, keep up the good work.

Side Note: One thing I think you should be conscious about with the multi-device support is reducing the computation as much as possible. It currently looks like everything is essentially being processed independently. For running on a desktop this isn't an issue, but my plan is to run this on a RaspberryPi that acts as the coordinator broadcasting to ESP8266s. The RaspberryPi is already pretty close to being maxed out, so I would expect (have not tested) that using once as a controller for multiple strips wouldn't be feasible. Would be great to factor things out such the processing the audio and building visualization are separated as much as possible with most of the computation on the audio processing side. Thus, driving multiple LEDs doesn't require more and more computing power.

Thanks!

**Comment by not-matt** [comment]
Yes, I have considered multiprocessing this on a Pi, and will add that
functionality in if I think it's required. I'm running this on my Windows
PC but have a Pi I want to set up for this specially.
